src/opentok-screen-sharing.js

```diff
@@ -307,10 +309,10 @@
 
   };
 
-  var start = function () {
+  var start = function (props) {
```

Comment:
Can you add in the description what are the allowed options in `props`?
An example: [https://github.com/opentok/accelerator-screen-sharing-js/blob/master/src/opentok-screen-sharing.js#L427](https://github.com/opentok/accelerator-screen-sharing-js/blob/master/src/opentok-screen-sharing.js#L427)

Also, maybe we can rename this props as `publisherOptions`?

Reply:
changed to publisherOptions